services/httpd/handler.go

```diff
@@ -182,6 +181,9 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	h.statMap.Add(statRequestsActive, 1)
 	start := time.Now()
 
+	// Add version header to all InfluxDB requests.
+	w.Header().Add("X-InfluxDB-Version", h.Version)
```

Comment:
Can you switch this to canonical casing (`X-Influxdb-Version`) so it goes through the happy path when verifying canonical case?